components/molecules/modals/m-modal-size-guide.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="m-modal-size-guide">
+    <SfModal :visible="isVisible" @close="closeModal">
+      <SfHeading :title="$t('Size guide')" />
+      <AStatic :content="content" />
+      <SfButton
+        class="sf-button--full-width color-secondary"
+        @click="closeModal"
+      >
+        {{ $t("OK") }}
+      </SfButton>
+    </SfModal>
+  </div>
+</template>
+
+<script>
+import {
+  SfModal,
+  SfButton,
+  SfHeading
+} from '@storefront-ui/vue';
+import AStatic from 'theme/components/atoms/a-static';
+import i18n from '@vue-storefront/i18n';
+
+export default {
+  name: 'MModalSizeGuide',
+  components: {
+    AStatic,
+    SfModal,
+    SfButton,
+    SfHeading
+  },
+  data () {
+    return {
+      content: [
+        {
+          title: 'Privacy Policy',
```

Comment:
The below text is not possible to translate. In my opinion, we can move it to /resource/ directory as JSON, then the translation will be possible through creating a subdirectory with translated JSON (fe. /resource/de_DE/...json)

Reply:
You are right. I moved those long default texts to separate .json file in `/resource/default-content.json` that can be easily translated to other languages - similarly like it was done for `promoted_offers` or `instagram-images`.